bin/pakiti-client

```diff
@@ -487,7 +521,6 @@ sub init () {
 
 sub main () {
     my(%data, $report);
-
```

Comment:
There's a bunch of unnecessary new-line/blank-space changes above, please get rid of them.